utils.js

```diff
@@ -56,29 +57,11 @@ const PAYLOAD_KEYS = [
   'tab_center_expanded'
 ];
 
-function Stats() {
-  for (let key of PAYLOAD_KEYS) {
-    this[key] = 0;
+function sendPing(key) {
+  if (!PAYLOAD_KEYS.includes(key)) {
+    // console.log(`Could not find ${key} in payload keys.`);
```

Comment:
Not really sure what purpose this clause serves; just saving yourself from typos?